low-level/files/opt/seagate/sspl/prerequisites/setup_yum_repos.sh

```diff
@@ -0,0 +1,483 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+# *********************************************************
+#  Description:   Add common and sspl repos to yum.repos.d
+#  Purpose    :   To install dependencies and SSPL rpms
+#  Usage      :   ./setup_yum_repos.sh --help
+# *********************************************************
+
+
+set -eE
+
+LOG_FILE="${LOG_FILE:-/var/log/cortx/sspl/sspl-prereqs.log}"
+export LOG_FILE
+
+echo "***** Running $0 *****"
+
+tmpdir="/tmp/_cortx_sspl_prereqs_"
+mkdir -p $tmpdir
+
+tgt_build=
+cortx_deps_repo=
+epel_repo=
+url_sspl_repo=
+url_uploads_repo=
+cortx_commons_repo="/etc/yum.repos.d/cortx_commons.repo"
+cortx_platform_base_repo="/etc/yum.repos.d/cortx_platform_base.repo"
+cortx_platform_extras_repo="/etc/yum.repos.d/cortx_platform_extras.repo"
+third_party_epel_repo="/etc/yum.repos.d/3rd_party_epel.repo"
+sspl_repo="/etc/yum.repos.d/sspl.repo"
+sspl_uploads_repo="/etc/yum.repos.d/sspl_uploads.repo"
+bundled_release=false
+CORTX_BASE_URL="http://cortx-storage.colo.seagate.com/releases/cortx"
+
+# Repo url for sspl, sspl_uploads & in house built commons packages CentOS systems
+if grep -q "CentOS Linux release 7.8" /etc/*-release; then
+    url_local_repo_commons="$CORTX_BASE_URL/third-party-deps/centos/centos-7.8.2003/"
+    url_uploads_repo="$CORTX_BASE_URL/uploads/centos/centos-7.8.2003/"
+elif grep -q "CentOS Linux release 7.7" /etc/*-release; then
+    url_local_repo_commons="$CORTX_BASE_URL/third-party-deps/centos/centos-7.7.1908/"
+    url_uploads_repo="$CORTX_BASE_URL/uploads/centos/centos-7.7.1908/"
+fi
+
+# Repo url for in house built commons packages for RHEL systems
+url_local_repo_commons_rhel="$CORTX_BASE_URL/third-party-deps/rhel/rhel-7.7.1908/"
+
+# Repo url for in house built HA packages for RHEL systems
+#url_local_repo_rhel_ha="$CORTX_BASE_URL/rhel_local_ha/"
+
+# Repo url for Saltstack
+# url_saltstack_repo="https://repo.saltstack.com/py3/redhat/$releasever/$basearch/3000"
+
+function trap_handler {
```

Comment:
Keep log statement little different in 2 trap handlers, to be able to identify from log lines too. Also single trap handler code wont be enough, both has very similar code!